.github/workflows/examples-infineon.yaml

```diff
@@ -25,15 +25,16 @@ concurrency:
 
 jobs:
     infineon:
-        name: Infineon examples building
-        timeout-minutes: 140
+        name: Infineon examples
+        timeout-minutes: 200
```

Comment:
This is very long (over 3 hours). Is this really how long things take? Can we somehow optimize things?

Reply:
Reverted the change back to 140. Build does not take that long and mostly around 80 mins